content/js2/blocks/response/index.md

```diff
@@ -31,26 +31,26 @@ textarea.addEventListener("keyup", handleKeyup);
 
 Let's break down the arguments that are passed to `addEventListener`.
 
-- [`"keyup"`](https://developer.mozilla.org/en-US/docs/Web/API/Element/keyup_event) - this is the type of event
-- `handleKeyup` - the second argument is a function. It is a function that is called when an event occurs.
+- [`"keyup"`](https://developer.mozilla.org/en-US/docs/Web/API/Element/keyup_event) - this is the type of event we want to be notified about
+- `updateCharacterLimit` - the second argument is a function. It is a function that is called when an event occurs.
 
-In JavaScript, we can pass functions as arguments to other functions. In this case, we're passing a function `handleKeyup` to `addEventListener` as an input. We can think of this as saying: whenever a key is released on the `textarea` element, then the function `handleKeyup` will get called. Any code we want to run in response to the `keyup` event will need to be executed inside `handleKeyup`.
+In JavaScript, we can pass functions as arguments to other functions. In this case, we're passing a function `updateCharacterLimit` to `addEventListener` as an input. We can think of this as saying: whenever a key is released on the `textarea` element, then the browser will call the function `updateCharacterLimit`. Any code we want to run in response to the `keyup` event will need to be inside the `updateCharacterLimit` function.
```

Comment:
When were you planning on introducing `Array.map` and `Array.filter`? I was going to make a comparison here, but realised we haven't actually covered them yet. I don't think that's a problem, but it triggered the question for me :)

Reply:
I'm thinking that week 1 of JS3 will be the ideal place to introduce `map` as we're going to talk about mapping a list of data to a list of UI elements.

Reply:
Sounds good!